github-management/org-owners-guide.md

```diff
@@ -47,6 +47,21 @@ code in.
 It is easier to start new code in a Kubernetes organization than it is to
 transfer in existing code.
 
+## Licensing Guidance
+
+Section 11 of the [CNCF Charter] provides guidance on how to license code and
+documentation owned by the CNCF projects. However, it doesn't explicitly cover
+situations where a repo may have mixed content (such as repos that contain both
+code and documentation living alongside each other).
+
+For clarity, the following guidance has been provided:
```

Comment:
s/the following guidance has been provided/here is the guidance for kubernetes github repositories/

being very specific and avoiding the who provided guidance to whom etc question :)

Reply:
LGTM otherwise